app/uk/gov/hmrc/agentclientrelationships/services/FindRelationshipsService.scala

```diff
@@ -75,4 +75,42 @@ class FindRelationshipsService @Inject()(des: DesConnector, val metrics: Metrics
           }
         ))
       .map(_.collect { case Some(x) => x }.groupBy(_._1).mapValues(_.map(_._2)))
+
+  def getInactiveRelationshipsForClient(identifiers: Map[EnrolmentService, EnrolmentIdentifierValue])(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext): Future[Seq[InactiveRelationship]] =
+    Future
+      .sequence(
```

Comment:
I don't think this is necessary.
Could you do:
`identifiers.get(EnrolmentMtdIt.enrolmentService).map(_.asMtdItId).map()